samples/eShopLite/AppHost/Program.cs

```diff
@@ -32,6 +32,6 @@
 builder.AddProject<Projects.CatalogDb>("catalogdbapp")
        .WithReference(catalogDb);
 
-builder.AddProject<Projects.Aspire_Dashboard>("dashboard");
+builder.AddProject<Projects.Aspire_Dashboard>("aspire-dashboard");
```

Comment:
This is only for the local playground, right? Could you add a comment to that effect. Make it clearer to anyone reading our code (working on aspire or using it) that adding the dashboard isn't typical.

Reply:
Once we've finished with the bulk of the review I'll go through all the playgrounds and add a reference to the dashboard as well.